### PR TITLE
Update readme with brief information about this repo

### PR DESCRIPTION
**Pull request by zmiklank**
I was not really sure, what is the function of this repository, so I took a look and created a brief README.md.

However, I am not sure, if really all fmf plans here are used by the GitHub Actions, so please correct me, if this information is incorrect.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# sclorg-testing-farm
\ No newline at end of file
+# Testing Farm FMF testing plans
```
Comment (phracek) [inline README.md:1]: I would prefer to mention here something like:

`FMF testing plans for using it in Testing Farm`.

**Review by phracek — CHANGES_REQUESTED** [review]
Thanks for the PR!. Can you please rephrase it a bit more?

I would prefer also to mention in README.md also a list of plans.

**Comment by phracek** [comment]
See the content of directory `https://github.com/sclorg/sclorg-testing-farm/tree/main/plans`. But it can be done as a separate pull request.

**Comment by zmiklank** [comment]
@phracek, I have updated the PR as you wanted, thanks for the review.

However, I think, that the fedora.fmf should be updated, because it contains "centos7" strings. I thus didn't know what exactly should I write to the fedora column in the table.

Also, the description in the c9s fmf plan mentions only docker tests and not openshift tests, as is in, e.g., centos7 fmf plan. Is this correct?

File: README.md [hunk 2]
```diff
@@ -1 +1,17 @@
...
+This repository contains testing plans, which are used by the [sclorg](https://github.com/sclorg/) testing GitHub Actions running on Testing Farm infrastructure.
+
+Available testing plans:
+
+| testing plan            | description               |
+|  --------------         |------------               |
+| `betka-bot.fmf`         | Runs betka tests on Fedora|
+| `betka-fedmsg.fmf`      | Runs betka-fedmsg tests on Fedora      |
+| `c9s.fmf`               | Runs docker tests on CentOS Stream 9   |
+| `fedora.fmf`            | not sure, **TODO**        |
```
Comment (phracek) [inline README.md:12]: This will be used in the future for testing RHSCL containers on the Fedora host.

**Review by phracek — APPROVED** [review]

**Comment by phracek** [comment]
> However, I think, that the fedora.fmf should be updated, because it contains "centos7" strings. I thus didn't know what exactly should I write to the fedora column in the table.
> 

This will be used later on. For now, we have not prepared infrastructure. As soon as we will use sclorg/testing-farm-as-github-action@v1, this will be used.
It is requested from @frenzymadness .

> Also, the description in the c9s fmf plan mentions only docker tests and not openshift tests, as is in, e.g., centos7 fmf plan. Is this correct?

For now, we are testing only container tests. Not openshift tests yet ;)

**Comment by zmiklank** [comment]
Thank you for the explanations!

**Pull request merged**